### DISCUSSION TITLE
處在AI時代的我們，難辨Deepfake假影片的真偽

### DISCUSSION THREAD

**Discussion by ccathyy** [1101-GS3073-C/Python-Community-GS3073-C#76] [General]
日新月異的科技帶給我們許多便利，卻在我們不知不覺時變成另一種威脅!

資料來源: https://www.bnext.com.tw/article/65676/ai-deepfake-gan-story-mooly

**Comment by jim11261314** [comment]
我的想法是Deepfake可以重現一些影像
像是過去偉人的演講，可能有留下聲音
卻沒有留下影像，用deepfake可以讓歷史人物更生動的表現

**Comment by LanningYang** [comment]
這就是科技帶來的利與弊

**Comment by ghost** [comment]
以前也常常看到真偽辨識的消息 很害怕以後會有假人和真人分辨不出來的窘境
